platform-sdk/settings.gradle.kts

```diff
@@ -52,44 +54,23 @@ include(":swirlds-platform-core")
 
 include(":swirlds-cli")
 
-include(":platform-apps")
-
-project(":platform-apps").name = "swirlds-platform-apps"
-
-include(":swirlds-platform-apps:demos:CryptocurrencyDemo")
-
-include(":swirlds-platform-apps:demos:HashgraphDemo")
-
-include(":swirlds-platform-apps:demos:HelloSwirldDemo")
-
-include(":swirlds-platform-apps:demos:StatsDemo")
-
-include(":swirlds-platform-apps:tests:AddressBookTestingTool")
-
-include(":swirlds-platform-apps:tests:ISSTestingTool")
-
-include(":swirlds-platform-apps:tests:MigrationTestingTool")
-
-include(":swirlds-platform-apps:tests:PlatformTestingTool")
-
-include(":swirlds-platform-apps:tests:ConsistencyTestingTool")
-
-include(":swirlds-platform-apps:tests:StatsSigningTestingTool")
-
 include(":swirlds-benchmarks")
 
-include(":swirlds-unit-tests")
+include(":swirlds-test-framework", "swirlds-unit-tests/common/swirlds-test-framework")
 
-include(":swirlds-unit-tests:common:swirlds-test-framework")
+include(":swirlds-common-testing", "swirlds-unit-tests/common/swirlds-common-test")
```

Comment:
I named this `:swirlds-common-testing` instead of `swirlds-common-test` because the module dependencies plugin cannot handle the following situation:

Module `com.swirlds.common` maps to `swirlds-common/src/main` (Subproject A)
Module `com.swirlds.common.test` maps to `swirlds-common-test/src/main` (Subproject B)
Module `com.swirlds.common.test.fixtures` maps to `swirlds-common/src/testFixtures` (Subproject A)

That is, having the same name prefix one time pointing at a different subproject (Subproject B) and then pointing back at another source set in the same project (Subproject A).